doc/apple2.sgml

```diff
@@ -511,6 +511,109 @@ url="ca65.html" name="assembler manual">.
 </descrip><p>
 
 
+<sect1>Specifying file types for fopen<p>
+
+<descrip>
+
+  <tag>Explanation of File Types</tag>
+
+  ProDOS associates a file type and an auxiliary type with each file.
+  These type specifications are separate from the file's name, unlike
+  Windows and UNIX-like systems which use the file name's suffix (a.k.a.
+  extension) to specify the file type. For example, .exe, .doc, or .bat.
```

Comment:
@oliverschmidt Very much true. I recall I once wrote this as a response to some discussion that involved "extensions": “traditional” UNIX (or Amiga) way of recognising what the file contains, is… to check what the file contains (sic!). Strange? [...]